release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -3376,6 +3376,50 @@ This section will continue to be updated over time to provide notes on enhanceme
+
+* Before this update, when a Cluster Operator took a long time to upgrade, the Cluster Version Operator (CVO) did not report anything because it could not determine if the upgrade was still progressing or already stuck. With this release, a new unknown status is added for the failing condition in the status of the cluster version reported by the CVO to remind the cluster administrators to check the cluster. As a result, the administrators do not need to wait on a blocked Cluster Operator upgrade. (link:https://issues.redhat.com/browse/OCPBUGS-58452[OCPBUGS-58452])
+
+* Before this update, when you changed the order of selectors in the `ClusterRole` parameter for the `OperatorGroup` in Operator Lifecycle Management (OLM), unnecessary etcd writes and auth cache invalidation degraded performance. With this release, an update to OLM prevents unnecessary etcd writes and auth cache invalidation when you change the selector order in the `ClusterRole` parameter. (link:https://issues.redhat.com/browse/OCPBUGS-58881[OCPBUGS-58881])
+
+* Before this update, when a `Machine Set` was scaled down and had reached its minimum size, the Cluster Autoscaler could leave the last remaining node with a no schedule taint that prevented use of a node. This issue was caused by a counting error in the Cluster Autoscaler. With this release, the counting error has been fixed so that the Cluster Autoscaler works as expected when a `Machine Set` is scaled down and has reached its minimum size. (link:https://issues.redhat.com/browse/OCPBUGS-59267[OCPBUGS-59267])
+
+* Before this update, the resources containing the configuration for the vSphere connection would break because of a mismatch between the user interface and the API. With this release, the resources do not break because the user interface uses the updated API definition. (link:https://issues.redhat.com/browse/OCPBUGS-60175[OCPBUGS-60175])
+
+* Before this update, the image registry would, in some cases, panic when attempting to purge failed uploads from s3-compatible storage providers. This issue was caused by the s3 driver for the image registry mishandling empty directory paths. With this release, the image registry properly handles empty directory paths, which fixes the panic. (link:https://issues.redhat.com/browse/OCPBUGS-60183[OCPBUGS-60183])
+
+* Before this update, insufficient obfuscation of new network data types in Hosted-Hypershift exposed user data. As a consequence, sensitive information was not protected. With this release, obfuscation for new network data types is implemented. As a result, obfuscated network data in Hosted-Hypershift improves data privacy. (link:https://issues.redhat.com/browse/OCPBUGS-60520[OCPBUGS-60520])
```

Comment:
```suggestion
* Before this update, insufficient obfuscation of new network data types in hosted control planes exposed user data. As a consequence, sensitive information was not protected. With this release, obfuscation for new network data types is implemented. As a result, obfuscated network data in hosted control planes improves data privacy. (link:https://issues.redhat.com/browse/OCPBUGS-60520[OCPBUGS-60520])
```
"Hosted-Hypershift" is not a thing, as far as I know. I believe this should be "hosted control planes".